bitnami/fluentd/README.md

```diff
@@ -216,8 +216,8 @@ The following tables lists the configurable parameters of the fluentd chart and
 | `metrics.serviceMonitor.annotations`            | ServiceMonitor annotations                                                                                     | `nil`                                                                                                   |
 | `tls.enabled`                                   | Enable the addition of TLS certificates                                                                        | `false`                                                                                                 |
 | `tls.caCertificate`                             | Ca certificate                                                                                                 | Certificate Authority (CA) bundle content                                                               |
```

Comment:
This parameter doesn't exist

Reply:
It seems it was removed by mistake, so it is fine maintaining it in the README, we should add it to the values